### PR TITLE
Todo list CRUD features

### PR DESCRIPTION
**Pull request by chukwuebukaVictor**
**Todo-list milestoe 2**

- Added edit and delete functionality.
- Implemented the add functionality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 > Todo list is a webapp where user can maintain a list of their todos.
 
-<img width="563" alt="Screenshot 2022-02-09 at 2 11 20 AM" src="./src/todoshot.png">
+<img width="563" alt="Screenshot 2022-02-09 at 2 11 20 AM" src="./src/todoshoot1.png">
 
 
 ## Built With
```
Comment (oloomoses) [inline README.md:10]: - [ ] To make the readme more descriptive, kindly add instructions on how to set up and run the app locally.
- [ ] The live link provided in the readme is not working and therefore, I would advise that you remove it.

File: src/modules/addToUI.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    textInput.addEventListener('keydown', (e) => {
+      text.innerHTML = textInput.value;
+      const index = parseInt(listItem.id, 10);
+      ToDo.list[index].description = text.innerHTML;
+      localStorage.setItem('todoList', JSON.stringify(ToDo.list));
+      if (e.code === 'Enter') {
+        text.style.display = 'block';
+        textInput.classList.toggle('edit-item');
+      }
```
Comment (oloomoses) [inline src/modules/addToUI.js]: - [ ] Kindly ensure that the edited tasks are updated in localStorage. When I edit the task and refresh the page, the original tasks is still displayed which means the localStorage is not updated.

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi @chukwuebukaVictor,

**Highlights**
- Highly organized/clean code :heavy_check_mark: 
- UI Design :heavy_check_mark: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly use the correct linter setup and configuration so that the checks can be performed by GitHub. You should have the `linters.yml` file inside [.github/workflows](https://github.com/microverseinc/linters-config/tree/master/javascript/.github/workflows)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @chukwuebukaVictor, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**